manifest/provider/apply.go

```diff
@@ -192,9 +192,7 @@ func (s *RawProviderServer) ApplyResourceChange(ctx context.Context, req *tfprot
 			if !isComputed {
 				return v, nil
 			}
-			if v.IsKnown() {
```

Comment:
This could be causing all the errors to show up, the reason for removing this is because all computedfields are now set to unknown, a value will never be known making this check pointless.

Reply:
This comment can be ignored. Issue is due to the new test that i added:

```
=== RUN   TestKubernetesManifest_ComputedFields
    computed_attr_test.go:85: Could not find resource "kubernetes_manifest.deployment_resource_diff" in state
--- FAIL: TestKubernetesManifest_ComputedFields (25.12s)
```

Reply:
Values in the manifest could be unknown even without being computed. For example, when being set from attributes of upstream resources that are only known after applying that resource. 

But regardless of that, this statement is where we skip over all the known values. Removing this line would prevent known values from being left in the manifest after the transform.

Reply:
Never mind, I think we're good removing this since the known values and non-computed unknown will be handled by the `if` above. Good catch!

Reply:
Passing Test has been added that checks when a Deploment value is changed by the API. Ready to Merge now! 🚀